packages/viewer/README.template.md

```diff
@@ -38,11 +38,35 @@ file that references our published JS bundles from a CDN.
   </head>
 
   <body>
-    <vertex-viewer id="viewer" src="urn:vertexvis:stream-key:123"></vertex-viewer>
+    <vertex-viewer id="viewer" src="urn:vertexvis:stream-key:123" client-id="123"></vertex-viewer>
   </body>
 </html>
 ```
 
+---
+
+This package also provides a set of utilities for use with its API.
+These utilities can be imported from a CDN as shown below:
+
+```html
+<!-- CDN -->
+<html>
+  <head>
+  </head>
+  <body>
+    <script type="module">
+      import { ColorMaterial } from 'https://unpkg.com/@vertexvis/viewer@{{version}}/dist/viewer/viewer.esm.js';
```

Comment:
`https://unpkg.com/@vertexvis/viewer@{{version}}/dist/esm/viewer.mjs`

Reply:
Currently the ESM bundle is still located at `/dist/viewer/viewer.esm.js`, do we want to update the location for this PR?

Reply:
Updated this to `https://unpkg.com/@vertexvis/viewer@{{version}}/dist/esm/index.mjs`